File: 백준/Bronze/1546. 평균/평균.py
# 다 더하고 최대값으로 나누고 갯수로 나누고 100곱함

n = int(input())
array = list(map(int,input().split()))
x = max(array)
y = sum(array)
z = (y/(x*n))*100

print(z)

File: 백준/Bronze/1546. 평균/평균.kt
import java.util.Scanner

fun main() = with(Scanner(System.`in`)) {
    var n = nextInt()
    var score = IntArray(n)
    var sum:Double = 0.0

    for (i in 0 until n) {
        score[i] = nextInt()
    }

    val maxScore = score.max()

    for (i in 0 until n) {
        var cheat:Double = (score[i].toDouble()/maxScore.toDouble())*100
        sum = sum + cheat
    }

    print(sum/n)
}